e2e/cypress/tests/integration/account_settings/profile/email_spec.ts

```diff
@@ -10,31 +10,47 @@
 // Stage: @prod
 // Group: @account_setting
 
+import {AdminConfig} from '@mattermost/types/lib/config';
+import {UserProfile} from '@mattermost/types/lib/users';
+
 import * as TIMEOUTS from '../../../fixtures/timeouts';
 import {reUrl, getRandomId} from '../../../utils';
 
 describe('Profile > Profile Settings > Email', () => {
     let siteName;
-    let testUser;
+    let testUser: UserProfile;
     let otherUser;
     let offTopicUrl;
+    let origConfig: AdminConfig;
```

Comment:
These types were already under [Cypress namespace](https://github.com/mattermost/mattermost-webapp/blob/master/e2e/cypress/tests/support/index.d.ts#L7-L34). With that, should we use it instead and remove the import declaration?
```javascript
let testUser: Cypress.UserProfile;
...
let origConfig: Cypress.AdminConfig;
```